### PR TITLE
HW1

### PR DESCRIPTION
**Pull request by asharnadeem**

### CODE REVIEW

File: an3056.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Open Source Development
+
+Name: Ashar Nadeem
+
+UNI: an3056
+
+GitHub: [link](https://github.com/asharnadeem)
+
+GitHub README: [link](https://github.com/asharnadeem/asharnadeem/blob/main/README.md)
```
Comment (timkpaine) [inline an3056.md:9]: From the homework: 
```Note Make sure this pull request is separate from your HW0 pull request, using a different branch if necessary```

Comment (timkpaine) [inline an3056.md:9]: From the homework: 
```Note Make sure this pull request is separate from your HW0 pull request, using a different branch if necessary```

**Review by timkpaine — CHANGES_REQUESTED** [review]

File: an3056.md (hunk 1 above)
Comment (timkpaine) [reply an3056.md:9]: xref: https://github.com/ColumbiaOSS/project-proposals-s2023/pull/68

Comment (asharnadeem) [reply an3056.md]: Addressed via branch switching

**Comment by timkpaine** [comment]
lgtm

**Pull request closed without merging**